minikube/tb-redis-configmap.yml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright © 2016-2020 The Thingsboard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tb-redis-config
```

Comment:
I suggest a tb-cache-config name
See an example on our QA cluster
the cache related env is not only about redis

```
  #CACHE
  #  CACHE_MAXIMUM_POOL_SIZE: "50" # default 16 # max pool size to process futures for the external cache
  #  TS_KV_PARTITIONS_MAX_CACHE_SIZE: "2000000" # default 100000 # helps to avoid excessive writes to Cassandra if partitioned
```